File: 800/200B.c
//Aadityaraj Kaushal going God Mode here </>
//https://codeforces.com/problemset/problem/200/B

#include <stdio.h>
#include <stdlib.h>





int main(){
    int n;
    scanf("%d", &n);

    int* p = (int*) malloc (sizeof(int) * n);
    float sum=0;
    for(int i=0 ; i<n ; i++){
        scanf("%d", p+i);
        sum += p[i];
    }

    printf("%f\n", sum/n);

    return 0;
}

File: python/800/200B.py
"""
    Author : Aadityaraj Kaushal
    Published : 04.03.2025
    Link : https://codeforces.com/problemset/problem/200/B
"""

n = int(input())
pi = input().strip().split(" ")

sum_pi = float(0)

for p in pi :
    sum_pi += float(p)

print(sum_pi/n)